### DISCUSSION TITLE
NFT withdrawal delay 30-90 days after blocking

### DISCUSSION THREAD

**Discussion by YamiYamiKokiKopi** [sunflower-land/sunflower-land#2569] [Ideas]
Hello.
I propose to remove the ban on the withdrawal of nft items after blocking the account. But do it not immediately, but after 30-90 days. This is how we fight against multi-account, because it will not be profitable for dishonest players to participate. But the big player will know that even if he is banned by mistake, he will keep his money.

p.s. I don't have a blocked account. But I sold t3 beaver and t3 kuyobiko, purely out of fear of a possible ban and loss of all resources.
I use VPN a lot because Twitter is banned in my country and I need it and I often use VPN in SFL and I have a strong fear that because of this I will be blocked.

I hope the team and the community will support this

**Comment by future2back** [comment]
Has any player ever been banned by mistake?

You will never be banned by mistake and if you are, you can open a ticket. Any game works like that.

**Reply by YamiYamiKokiKopi** [reply]
Why do we need fear?
Do you think this position will help attract more players with more investments?
A very illogical answer.

If for example I get banned and unbanned in 7 days and I have 3k+ dollars in NFT on the farm. Do you think after my unban I won't sell them? I will be afraid of the ban in the future.

Well, you say that there are no problems. But I sold 2 nft t3. Are you worried about my fears?
And so I will say that I know 3 people who have a similar problem. I don’t know how widespread this fear is, but it exists and has not been resolved for a year now. I would like to see any suggestion from the team or the community, maybe even in a different way.

I will add:
Do not give the output of the old nft. Allow only those who were banned after the update to display nft

**Comment by VZozul** [comment]
To be honest, I also had thoughts of selling the Iron Idol for the same reason. Although I have no reason to ban, but sometimes in Discord you read how someone was banned, he swears that he did not use any third-party programs, but his account was rechecked and they saw the opposite there. For some reason I believe him that he did not use anything. And this fear that you will be banned and there is nothing you can do about it. Let's say I play a lot, I collect resources even at night, all by alarm clocks. Actions can be considered the same type. Here is the ban, and try to prove that you collect it yourself.
But! Allowing withdrawals from a banned account is not an option, you can't do that, where have you seen this in other games? Some other option is needed. And it's really hard to come up with it. You've been jailed for causing damage to the game, followed by the seizure of property that goes to the game. Or, depending on the damage, you compensate for the damage and sit in jail for your term, then you get out and already do what you want with your SFTs. That is, you get a ban for a certain period and pay a fine. Now, if there were statistics of bans, how many of them are there in the game. Then one would think, if there are a lot of them, then a support service is needed to understand and pay attention to each ban, to understand in detail. Then you can enter in case of controversial cases not a ban, but simply a ban on the withdrawal of everything! Keep playing, but for 90 days you will not be able to withdraw anything, and after 90 days your account will be reviewed what exactly you did these 90 days, how it differed from what you did before. It’s just that if we assume this is a multi-account that the program collected resources, then this is also visible, and in these 90 days it simply won’t play, it will just wait for it to be unbanned.
For example, even if someone uses some autoclickers, when viewing his account, they will see that he has not withdrawn a penny from the game, and even vice versa. Will he get a ban? I think not, this is called an individual approach and analysis of each case.

**Reply by adamhannigan** [reply]
That is fair feedback.

When it comes to multi-account detection it is extremely hard to discern who the real players are since every multi-accounter/bot that has been found guarantees they were not doing it.

To be honest, we have had probably 20-30 of these cases where real players did get identified by Jigger who were not multiaccounting, just helping friends and family.

In these cases we've always taken time to do personal investigations, talk with the player and lift the ban. It is frustrating, but these occurences are usually resolved in a week or so. ❤️

**Reply by Noonran** [reply]
hello adam, 
this was my case, 
beginner in web3 I wanted to take an account for my wife and her son by sharing my wallet on his smartphone, but realizing my mistake I reported the thing on discord then to moderator (you can check ticket and vipchat @meta_noon), they explained what to do but in the end I was banned.
no matter how hard I tried to explain my situation, the technical team wouldn't listen, taking me for someone dishonest, even though I was active on discord and even nitro booster, I believed it....
On this farm, which was my pride and joy, there were more than 2 eth items, my very first investment in nft, which left me with a bitter taste.
Not one to hold a grudge, I took over another account and continued for a while...
